backend/pkg/commons/db/migrations/postgres/TBD_add_notifications_column.sql

```diff
@@ -0,0 +1,13 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query - add column notifications_do_not_disturb_ts and change event_threshold data type';
+ALTER TABLE users ADD COLUMN IF NOT EXISTS notifications_do_not_disturb_ts TIMESTAMP WITHOUT TIME ZONE;
+ALTER TABLE users_subscriptions ALTER COLUMN event_threshold SET DATA TYPE NUMERIC;
+-- +goose StatementEnd
+
+-- +goose Down
+-- +goose StatementBegin
+SELECT 'down SQL query - remove column notifications_do_not_disturb_ts and revert event_threshold data type';
+ALTER TABLE users DROP COLUMN IF EXISTS notifications_do_not_disturb_ts;
+ALTER TABLE users_subscriptions ALTER COLUMN event_threshold SET DATA TYPE REAL;
+-- +goose StatementEnd
```

Comment:
reminder to coordinate that this gets applied to user db (if not done yet)

Reply:
Sorry, I forgot to delete this file.
The desired behaviour in the db was discussed in BEDS-483 and it will also implement it.

We will leave the threshold as REAL(float32) and I also handled everything in the code as a float.
The Gas thresholds that we get as Wei (decimal.Decimal) get converted with `.Div(1e9).InexactFloat64()`.